### DISCUSSION TITLE
v4.5.0 (Beta)

### DISCUSSION THREAD

**Discussion by osy** [utmapp/UTM#6162] [Releases]
## Highlights
* **UTM Remote server for macOS** On macOS 13+, you can enable UTM Server from the new option on the home screen or from Window -> UTM Server. Once enabled, you can stream QEMU backend VMs to supported clients. The preferences page includes additional options including auto-starting the server and allowing external connections so it can be used outside of the local network. New documentation pages will be added in the future.
* **UTM Remote client for iOS and visionOS** When UTM Server is enabled on macOS, you can connect to it with the new UTM Remote client which uses the same frontend as UTM for iOS/visionOS but without any of the QEMU backend. TestFlight will be available as soon as it is approved and the plans are to release it as a free app in the App Store.

## Changes (v4.5.0)
* Updated ANGLE to latest Safari version
* Fixed a crash due to screenshot being saved while the image was being destroyed (#4009)
* Fixed a memory leak caused by a retain cycle while observing changes in the VM state
* Localization: Added Italian (thanks @garamb1)
* Localization: Updated Japanese (thanks @MMP0)
* Localization: Updated Polish (thanks @mavethee)
* Localization: Updated Chinese (Simplified + Traditional) (thanks @changanmoon)
* (macOS) Add UTM Server feature
* (iOS) Fixed the icon name for UTM SE erroneously set to UTM in translation (#5968)
* (iOS) Disable logging of keyboard keys when Debug Logs are enabled. They aren't helpful for any issue debugging and poses a privacy issue.
* (iOS) Reworked automatic resolution changing when SPICE tools are installed
* (iOS) Fix crash due to race when re-sizing while a view is being destroyed
* (visionOS) Move toolbar to top of window and support hiding
* (visionOS) Make VM windows less rounded
* (visionOS) Integrate new keyboard view

## Issues
Please check the full list on [Github](https://github.com/utmapp/UTM/issues) and help report any bug you find that is not listed.

## Installation

* [iOS](https://docs.getutm.app/installation/ios/)
* [macOS](https://docs.getutm.app/installation/macos/)

| File | Description | Installation | JIT | Hypervisor | USB |
|------|------------|--------------|-----|-----------|-----|
| UTM.deb | Jailbroken iOS version | Open in Cydia, dpkg, or Sileo | Yes | Yes(1) | Yes |
| UTM.dmg | macOS version | Mounting and copying UTM.app to /Applications | Yes | Yes | Yes |
| UTM.ipa | Non-jailbroken iOS version (sideloading) | AltStore, etc (see guide) | Yes(2) | No | No |
| UTM-HV.ipa | Non-jailbroken iOS version (TrollStore) | TrollStore | Yes | Yes(1) | Yes |
| UTM-SE.ipa | Non-jailbroken iOS version (sideloading) | AltStore, enterprise signing, etc | No | No | No |
| UTM-Remote.ipa | Remote client | Any | No | No | No |

1. Hypervisor on iOS requires an M1 iPad.
2. Enabling JIT may require a separate JIT enabler such as [Jitterbug][2] or Jitstreamer.

[1]: https://getutm.app/install/
[2]: https://github.com/osy/Jitterbug

<hr /><em>This discussion was created from the release <a href='https://github.com/utmapp/UTM/releases/tag/v4.5.0'>v4.5.0 (Beta)</a>.</em>

**Comment by flanter21** [comment]
Which version of virgl does this use? Is it new enough to support [Windows 3D acceleration with the correct drivers](https://gitlab.freedesktop.org/virgl/virglrenderer/-/merge_requests/1185) as mentioned previously in https://github.com/utmapp/UTM/discussions/6035

**Comment by jifbrodeur** [comment]
I do not see the URL to download the the beta version 4.5.0.

**Reply by flanter21** [reply]
https://github.com/utmapp/UTM/releases/tag/v4.5.0

**Reply by jifbrodeur** [reply]
Thank you very much I see it now :)
